docker-compose.dev.core.yaml

```
@@ -1,156 +1,121 @@
-        - GH_TOKEN
-    env_file:
-      - env/event-flow.env
-      - .env
-    environment:
-      - CONFIGURATION_DATABASE_USER=postgres
-      - CONFIGURATION_DATABASE_PASSWORD=unused
-    restart: always
-    depends_on:
-      - postgres
-      - valkey
+# base
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)